docs/06-litellm-gateway.md

```
@@ -0,0 +1,756 @@
+- Terminate TLS via Cloudflare Tunnel (no in-cluster TLS termination needed)
+
+### Non-Functional Requirements
+
+- Availability: best-effort (homelab); no SLA
+- Latency: p50 proxy overhead <100ms
+- Scale: 1 replica; <10 concurrent requests typical
+- Cost: Mistral API pay-per-token; Anthropic API pay-per-token; no fixed infra cost beyond k3s node
+
+### Component Architecture
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)